src/md/release.md

```diff
@@ -156,6 +156,14 @@ svn import svn-dist https://dist.apache.org/repos/dist/dev/apex/${RNAME}-RC1 -m
 
 ## Build and Deploy Documentation
 
+### Javadoc
```

Comment:
@tweise @amberarrow Is there a way to build java doc for the release and not for the SNAPSHOT version? Both https://ci.apache.org/projects/apex-core/apex-core-javadoc-release-3.4/ and https://ci.apache.org/projects/apex-core/apex-core-javadoc-release-3.5/ use unreleased snapshot version.